projects/dot-rules/src/lib/styles/rule-engine.scss

```diff
@@ -94,6 +94,15 @@ $finder-s-input: '.ui.input input';
         height: 100%;
     }
 
+    .close-button {
+        font-size: 14px;
```

Comment:
![image](https://user-images.githubusercontent.com/37185433/77176136-42ae3b80-6a89-11ea-825b-5917e1e08874.png)

shouldn't you create a var for that?

Reply:
Not sure if we can have variables here.

Reply:
done https://github.com/dotCMS/core-web/pull/1289/commits/4df18ab78c2446282bbbd4d0bcf781cf39d4d912#diff-4a35aea585a8446a8a175a9ec9a43f72R99